codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameWriter.java

```diff
@@ -509,18 +511,23 @@ private ChannelFuture writeContinuationFrames(ChannelHandlerContext ctx, int str
                 ctx.write(fragment, promiseAggregator.newPromise());
 
                 // Write out the padding, if any.
-                if (padding > 0) {
-                    ctx.write(ZERO_BUFFER.slice(0, padding), promiseAggregator.newPromise());
+                if (paddingBytes(padding) > 0) {
+                    ctx.write(ZERO_BUFFER.slice(0, paddingBytes(padding)), promiseAggregator.newPromise());
                 }
             } while(headerBlock.isReadable());
         }
         return promiseAggregator;
     }
 
-    private static void writePaddingLength(ByteBuf buf, int paddingLength) {
-        if (paddingLength > 0) {
+    private static int paddingBytes(int padding) {
```

Comment:
Probably should add a bit of javadoc to make it clear how it is different from `padding`. The implementation is "obvious" in what it does, but not quite _why_ it does.